src/main/java/com/google/devtools/build/bfg/Bfg.java

```diff
@@ -108,6 +111,7 @@ private void run(String[] args) throws Exception {
     cmdLineParser.parseArgument(args);
 
     List<String> dotFile = readDotFileFromStdIn();
+    ParserOutput parserOutput = ParserOutput.parseFrom(System.in);
     if (dotFile.isEmpty()) {
```

Comment:
remove? 

what arguments are you testing this against? 
From what I understand of the change, there will be no more dotfiles, and also no more jdeps. Consequently, this should result in error.

Reply:
Done.